scarb/src/ops/package.rs

```diff
@@ -166,14 +173,44 @@ fn package_one_impl(
         check_metadata(pkg, ws.config())?;
     }
 
-    let recipe = prepare_archive_recipe(pkg, opts, ws)?;
+    let mut recipe: Vec<ArchiveFile> = Vec::default();
+
+    if let Some(script_definition) = pkg.manifest.scripts.get("package") {
+        let target_dir_path = target_dir.path_existent()?;
+        ws.config().ui().print(Status::new(
+            "Running package script with package",
+            &pkg_id.to_string(),
+        ));
+
+        if let Some(includes) = pkg.manifest.metadata.include.clone() {
+            copy_items_to_target_dir(includes, target_dir_path)?;
+        }
+
+        execute_script(script_definition, &[], ws, target_dir_path, None)?;
```

Comment:
Why are we executing the script in `target/package`?

It seems to me it should be either executed in the root dir, or if we want avoid modifying any unnecessary files there, in `target/package/<name>-<version>`?

Originally posted in: https://github.com/software-mansion/scarb/pull/1789#discussion_r1877804646